js/sykmeldinger/data/din-sykmelding/dinSykmeldingSagas.js

```diff
@@ -56,19 +56,46 @@ const getSykmeldingerBackendUrl = () => {
 
 export function* bekreftSykmelding(action) {
     yield put(actions.bekrefterSykmelding());
+    const url = `${getSykmeldingerBackendUrl()}`;
     try {
         const { sykmeldingId, verdier } = action;
+        const sporsmalOgSvarListe = [{
+            tekst: 'Jeg er sykmeldt fra',
+            shortName: 'ARBEIDSSITUASJON',
+            svartype: 'ARBEIDSSITUASJON',
+            svar: verdier.arbeidssituasjon,
+        }];
+        if (erFrilanserEllerSelvstendig(verdier)) {
+            if (verdier.harForsikring) {
+                sporsmalOgSvarListe.push({
+                    tekst: 'Har du forsikring som gjelder de første 16 dagene av sykefraværet?',
+                    shortName: 'FORSIKRING',
+                    svartype: 'JA_NEI',
+                    svar: verdier.harForsikring ? 'JA' : 'NEI',
+                });
+            }
+            if (verdier.harAnnetFravaer) {
+                sporsmalOgSvarListe.push({
+                    tekst: 'Brukte du egenmelding eller noen annen sykmelding før datoen denne sykmeldingen gjelder fra?',
+                    shortName: 'FRAVAER',
+                    svartype: 'JA_NEI',
+                    svar: verdier.harAnnetFravaer ? 'JA' : 'NEI',
+                });
+            }
+            if (verdier.egenmeldingsperioder) {
+                sporsmalOgSvarListe.push({
+                    tekst: 'Hvilke dager var du borte fra jobb før datoen sykmeldingen gjelder fra?',
+                    shortName: 'PERIODE',
+                    svartype: 'PERIODER',
+                    svar: verdier.egenmeldingsperioder,
+                });
+            }
+        }
         const body = {
-            feilaktigeOpplysninger: verdier.feilaktigeOpplysninger,
-            arbeidssituasjon: verdier.arbeidssituasjon,
-            harForsikring: erFrilanserEllerSelvstendig(verdier)
-                ? verdier.harForsikring : null,
-            harAnnetFravaer: erFrilanserEllerSelvstendig(verdier)
-                ? verdier.harAnnetFravaer : null,
-            egenmeldingsperioder: erFrilanserEllerSelvstendig(verdier)
-                ? verdier.egenmeldingsperioder : null,
+            timestamp: new Date().toUTCString(),
```

Comment:
Jeg ønsker å få et timestamp med UTC-format. Dette er kanskje ikke riktig måte å gjøre det på..?

Reply:
`new Date().toISOString()` gjør det du vil :)

Reply:
Feature-request github: Oppdatere siden når noen kommenterer og ikke vente på at jeg refresher 😛  Takk for tipset!